Scripts/script-CommonIntegration.yml

```diff
@@ -19,6 +19,15 @@ script: |-
           }
           return query;
   };
+  
+  function rplaceInTemplates(input, args) {
```

Comment:
Is this being used in an integration somewhere? We need to fix the typo there as well..